### PR TITLE
USHIFT-2123: Use dnf_retry.sh script when installing RPM packages

### PR DESCRIPTION
**Pull request by ggiguash**

### CODE REVIEW

File: scripts/image-builder/configure.sh [hunk 1]
```diff
@@ -31,21 +34,20 @@ else
...
+     containernetworking-plugins expect"
 
 sudo systemctl enable osbuild-composer.socket --now
 sudo systemctl enable cockpit.socket --now
 sudo firewall-cmd --add-service=cockpit --permanent
 
 # The mock utility comes from the EPEL repository
-sudo dnf install -y "https://dl.fedoraproject.org/pub/epel/epel-release-latest-${OSVERSION}.noarch.rpm"
-sudo dnf install -y mock nginx tomcli parallel
+"${DNF_RETRY}" install "https://dl.fedoraproject.org/pub/epel/epel-release-latest-${OSVERSION}.noarch.rpm"
```
Comment (eslutsky) [inline scripts/image-builder/configure.sh:49]: for readability i would also put the 2nd parameter  in  apostrophes "${DNF_RETRY}" "install"

**Review by eslutsky — COMMENTED** [review]
small change .otherwise lood good.

File: scripts/image-builder/configure.sh (hunk 1 above)
Comment (ggiguash) [reply scripts/image-builder/configure.sh]: Done

**Pull request merged**